assets/css/default-calendar-common.css

```
@@ -0,0 +1,459 @@
+	-moz-osx-font-smoothing: grayscale;
+}
+
+.simcal-icon-spin {
+	-moz-animation: spin 2s infinite linear;
+	-o-animation: spin 2s infinite linear;
+	-webkit-animation: spin 2s infinite linear;
+	animation: spin 2s infinite linear;
+	display: inline-block;
+}
+
+@-moz-keyframes spin {
```

Stylelint (17.14.0)

[error] Vendor-prefixed at-rule "`@-moz-keyframes`" (at-rule-no-vendor-prefix)

(at-rule-no-vendor-prefix)